.azure-pipelines/build-job.yml

```diff
@@ -116,6 +116,12 @@ jobs:
       env:
         ADO_ENABLE_LOGISSUE: true
 
+  - ${{ if or(eq(parameters.os, 'linux'), eq(parameters.os, 'osx')) }}:
```

Comment:
Just checking -- should this change come along?

Reply:
Revert it back.
Fixed it in another [PR](https://github.com/microsoft/azure-pipelines-agent/pull/4110)